diff --git a/node_modules/node-internet-radio/lib/icystream.js b/node_modules/node-internet-radio/lib/icystream.js
index f481aa0..7ee0988 100644
--- a/node_modules/node-internet-radio/lib/icystream.js
+++ b/node_modules/node-internet-radio/lib/icystream.js
@@ -51,7 +51,7 @@ function getStreamStation(url, callback) {
     var client = tls.connect(
       port,
       url.hostname,
-      { ecdhCurve: false, servername: url.hostname },
+      { servername: url.hostname },
       function () {
         client.write(getString);
       }
